openvdb_houdini/houdini/Utils.h

```diff
@@ -305,10 +305,45 @@ bool isLogForwarding(OP_OpTypeId);
 /// @}
 
 
+////////////////////////////////////////
+
+
+// Grid type lists, for use with openvdb::GridBase::apply()
+
+using ScalarGridTypes = openvdb::TypeList<
+    openvdb::BoolGrid,
+    openvdb::FloatGrid,
+    openvdb::DoubleGrid,
+    openvdb::Int32Grid,
+    openvdb::Int64Grid>;
+
+using NumericGridTypes = openvdb::TypeList<
+    openvdb::FloatGrid,
+    openvdb::DoubleGrid,
+    openvdb::Int32Grid,
+    openvdb::Int64Grid>;
+
+using RealGridTypes = openvdb::TypeList<
+    openvdb::FloatGrid,
+    openvdb::DoubleGrid>;
+
+using Vec3GridTypes = openvdb::TypeList<
+    openvdb::Vec3SGrid,
+    openvdb::Vec3DGrid,
+    openvdb::Vec3IGrid>;
+
+using PointGridTypes = openvdb::TypeList<
+    openvdb::tools::PointIndexGrid,
+    openvdb::points::PointDataGrid>;
+
+using NonPointGridTypes = ScalarGridTypes::Append<Vec3GridTypes>;
```

Comment:
I think "VolumeGridTypes" would be better than "NonPointGridTypes".

Reply:
Yes, that's better.